src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,41 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String, name)
+    if %w[now() current_time current_timestamp].includes?(default.downcase)
+      return " DEFAULT NOW()"
+    end
+    return " DEFAULT '#{default}'"
+  end
+
+  def default_value(type : Time.class, default : (String | Time | Symbol), name)
+    if default.is_a?(Symbol) && default == :now
+      " DEFAULT NOW()"
+    elsif default.is_a?(String) && %w[now() current_timestamp].includes?(default.downcase)
+      " DEFAULT NOW()"
+    elsif default.is_a?(Time)
+      " DEFAULT '#{default.to_utc}'"
+    else
+      raise "Unrecognized default value for #{name}: #{default}. Please use :now, 'now()' or 'current_timestamp'"
+    end
+  end
+
+  def default_value(type : (Int32 | Int64).class, default : (Int32 | Int64), name)
+    if type == Int32 && default > Int32::MAX
```

Comment:
Instead of manually handling this error I think it would be better to get something at compile time by using two different signatures: `default_value(type : Int32.class)` and another for Int64

Reply:
I keep forgetting Crystal is compiled! I'll clean it up, thanks